Removed kubenet reference from docs (#31667)
kubernetes/website: content/en/docs/tasks/network/validate-dual-stack.md

* Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)

> * A [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports dual-stack (such as Calico or Cilium)

Reviewer: It's actually better not to list examples:
```suggestion
* A [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
  that supports dual-stack networking.
```

Author: done.
